src/go/k8s/tests/e2e/external-connectivity/00-assert.yaml

```diff
@@ -29,9 +29,9 @@ spec:
             - redpanda
             - start
             - --check=false
-            - --reserve-memory 1M
```

Comment:
this change is not needed right?

Reply:
The order changed due to:

- https://github.com/vectorizedio/redpanda/pull/1168/files#diff-11cd0daf9c52813e733b2526ffd440b85ff6e9a8f6a11d8454c02cbd129bcb60L337-L338
- https://github.com/vectorizedio/redpanda/pull/1168/files#diff-11cd0daf9c52813e733b2526ffd440b85ff6e9a8f6a11d8454c02cbd129bcb60R452